packages/myst-transforms/src/frontmatter.ts

```diff
@@ -64,6 +65,10 @@ export function getFrontmatter(
       // If this has a label what do we do? Add this label as a document reference
       (nextNode as any).type = '__delete__';
       frontmatter.content_includes_title = false;
+      if (nextNode.label) {
+        const { identifier } = normalizeLabel(nextNode.label) ?? {};
+        if (identifier) identifiers.push(identifier);
```

Comment:
🤔 Maybe we want to also push the html-id for the title, e.g. so `[](#introduction-to-my-research)` would also work...